guides/code-style/.rubocop.yml

```diff
@@ -48,10 +44,6 @@ Style/IndentArray:
 Style/MultilineMethodCallIndentation:
   EnforcedStyle: indented
 
-Style/NumericLiterals:
```

Comment:
Because I put the excludes in the general "AllCops" for others this cop enabled by default and it keep unnecessary place now. 
Confirmation that cop "On" by default - https://github.com/rubocop-hq/rubocop/blob/master/manual/cops_style.md#stylenumericliterals